tests/test_cli/test_analytics.py

```diff
@@ -110,3 +110,18 @@ def test_opt_in_subcommand(
     # Check the contents of the config file match
     with open(config_file_path) as f:
         assert dict(yaml.safe_load(f)) == expected_configuration
+
+
+def test_cli__analytics_command_no_args(runner):
```

Comment:
There's a double underscore in the function name and missing typing for the `runner`, it should be `runner: TemplateRunner`